### PR TITLE
Add support for list of values in sqlQQ

### PR DESCRIPTION
**Pull request by alexkazik**
Before submitting your PR, check that you've:

- [x] Bumped the version number
- [x] Documented new APIs with [Haddock markup](https://www.haskell.org/haddock/doc/html/index.html)
- [kinda] Added [`@since` declarations](http://haskell-haddock.readthedocs.io/en/latest/markup.html#since) to the Haddock

After submitting your PR:

- [x] Update the Changelog.md file with a link to your PR
- [x] Check that CI passes (or if it fails, for reasons unrelated to your change, like CI timeouts)

### CODE REVIEW

File: persistent/ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2.9.2
+
+* Added support for list of values in `sqlQQ`. [#819](https://github.com/yesodweb/persistent/pull/819)
```
Comment (bitemyapp) [inline persistent/ChangeLog.md]: 2.9.1 hasn't been released so you'd just add it underneath the 2.9.1 header.

**Comment by bitemyapp** [comment]
I'm iffy on this. Could you please add tests? I don't feel like I can evaluate an addition like this without test coverage.

**Comment by alexkazik** [comment]
I've added a test (and moved the change to 2.9.1)

**Comment by parsonsmatt** [comment]
@alexkazik Thanks for the test and PR - sorry it's been a while since the last review. Would you be willing to relocate the code changes? The QuasiQuote functionality was moved into the `persistent-qq` package. If you'd be willing to update the PR, I'd be happy to get it merged.

**Comment by alexkazik** [comment]
I've updated the PR.

Should the parenthesis be output by the qq ("x IN (%{vals})" or  "x IN %{vals}"?

Can you add persitent-qq to stackage?

**Comment by parsonsmatt** [comment]
I think I would prefer for the syntax to be `[sqlQQ| ... WHERE x IN %{vals} |]`, though I don't have a strong opinion on this. If the primary use case for this is to allow people to use `IN`, then that would be the most convenient. I can't think of another use for interpolating a comma-separated list of SQL values.

I'll make a separate issue for `persistent-qq` to be added to Stackage. Thanks!

**Comment by alexkazik** [comment]
Moved the parenthesis into the function (and also added dependencies for ghc < 8).

**Comment by parsonsmatt** [comment]
@alexkazik The CI was stale and not checking against new versions of code. I've canceled your PR to get the new CI improvements in faster. When those CI changes are done, please merge them in and re-push and I'll help get this PR in. Thanks for the help!

**Comment by parsonsmatt** [comment]
Alright, CI has been updated. If you're willing to merge master, then getting the CI working should be mostly straightforward :)

**Comment by alexkazik** [comment]
Now everything works just fine

**Comment by parsonsmatt** [comment]
Fantastic, thanks :)

**Comment by parsonsmatt** [comment]
@snoyberg You are the only one with push access on Hackage. I've bumped the `package.yaml` version on `master`. Would you be willing to either add me as a Hackage pusher or do the upload?

Thanks!

**Comment by snoyberg** [comment]
@parsonsmatt added on Hackage :)

**Comment by parsonsmatt** [comment]
Published, thanks! :smile:

**Pull request merged**